src/themes/DiamondTheme.ts

```diff
@@ -1,47 +1,86 @@
-import { createTheme, ThemeOptions } from '@mui/material/styles';
+import { createTheme, Theme } from "@mui/material/styles";
 
-import {BaseThemeOptions} from "./BaseTheme";
+import { BaseThemeOptions } from "./BaseTheme";
 
-const dlsLogoBlue = "#202740"
-const dlsLogoYellow = "#facf07"
+const dlsLogoBlue = "#202740";
+const dlsLogoYellow = "#facf07";
 
-const DiamondTheme: ThemeOptions = createTheme({
+const DiamondTheme: Theme = createTheme({
   ...BaseThemeOptions,
   colorSchemes: {
     // https://zenoo.github.io/mui-theme-creator/
     light: {
       palette: {
         primary: {
           main: dlsLogoBlue,
-          // contrastText: '#ffffff',
-          // dark: '#161B2C',
-          // light: '#4C5266',
+          light: "#4C5266", // dark grey
+          dark: "#161B2C", // dark blue
+          contrastText: "#ffffff", // white
         },
         secondary: {
-            main: dlsLogoYellow,
-            // light: '#FBD838',
-            // dark: '#AF9004',
-            // contrastText: '#000000',
-        }
+          main: dlsLogoYellow,
+          light: "#FBD838", // light yellow
+          dark: "#AF9004", // dark yellow
+          contrastText: "#000000", // black
+        },
       },
     },
     dark: {
       palette: {
         primary: {
-          main: '#6175bd',  //lightened version of {dlsLogoBlue}
-          // light: '#8090CA',
-          // dark: '#435184',
-          // contrastText: '#ffffff',
+          main: "#6175bd", //lightened version of {dlsLogoBlue}
+          light: "#8090CA", // lighter blue
+          dark: "#435184", // mid blue
+          contrastText: "#ffffff", // white
         },
         secondary: {
           main: dlsLogoYellow,
-          // light: '#FBD838',
-          // dark: '#AF9004',
-          // contrastText: '#000000',
-        }
+          light: "#FBD838", // light yellow
+          dark: "#AF9004", // dark yellow
+          contrastText: "#000000", // black
+        },
+      },
+    },
+  },
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        root: ({ theme }) => ({
+          borderBottom: "2px solid rgba(0,0,0,0.2)",
+          textTransform: "none",
+          "&.default": {
+            color: theme.palette.primary.contrastText,
+            backgroundColor: theme.palette.primary.dark,
+            "&:hover": {
+              backgroundImage: "linear-gradient(rgb(0 0 0/30%) 0 0)",
+              "&:disabled": {
+                backgroundColor: theme.palette.primary.light,
+              },
+            },
+          },
+          "&.onBlue": {
```

Comment:
I wonder if this was supposed to say "onBlur" !

Reply:
I think it's for when the background is (Diamond) blue.

Reply:
I talked with Guilherme and he think it's a typo in his original code.

Reply:
I didn't see it used anywhere with onBlur, I can change it to something more descriptive for how it's being used here.